portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,18 @@
 <!DOCTYPE html>
+<html>
+<!DOCTYPE html> 
 <html>
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body> 
+      <header id="showcase">
+          <h1>Siham Abdi</h1> 
+           <p>Programmer, Problem Solver, Student.</p>
+          <a href="second.html" class="button">Read More</a>
```

Comment:
Unless I'm missing something, you don't seem to have a "second.html" page in this commit or in your portfolio, so I would either remove this link or add the page with the additional content!

Reply:
Ah okay, I see that you have the second page in your second pull request. I think that's fine in this case then, feel free to ignore the above comment.